src/ng-app/assets/locale/lang.nl.xlf

```diff
@@ -0,0 +1,1403 @@
+      <trans-unit approved="yes" datatype="html" id="noFanSpeedValue">
+        <source>N/A</source>
+        <target phase-name="approval-1" state="signed-off">n/b</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/cpu-dashboard/cpu-dashboard.component.ts</context>
+          <context context-type="linenumber">224</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit approved="yes" datatype="html" id="gaugeTextOff">
+        <source>off</source>
+        <target phase-name="approval-1" state="signed-off">uit</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/cpu-dashboard/cpu-dashboard.component.ts</context>
+          <context context-type="linenumber">230</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit approved="yes" datatype="html" id="gaugeTextOn">
+        <source>on</source>
+        <target phase-name="approval-1" state="signed-off">aan</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/cpu-dashboard/cpu-dashboard.component.ts</context>
+          <context context-type="linenumber">232</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit approved="yes" datatype="html" id="36a9f3e32b06903ce9038e66286fe4c0e45f0559">
+        <source>System monitor</source>
+        <target phase-name="approval-1" state="signed-off">Systeemmonitor</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/cpu-dashboard/cpu-dashboard.component.html</context>
+          <context context-type="linenumber">25</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit approved="yes" datatype="html" id="0551549b43ce74879408c2bf92a8035bfa6ea54b">
+        <source>Main processor monitor</source>
+        <target phase-name="approval-1" state="signed-off">Hoofdprocessormonitor</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/cpu-dashboard/cpu-dashboard.component.html</context>
+          <context context-type="linenumber">35</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit approved="yes" datatype="html" id="f67ab12753f4365b36a78ac5b6d169538ec2af34">
+        <source>CPU - Temp</source>
+        <target phase-name="approval-1" state="signed-off">Cpu-temperatuur</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/ng-app/app/cpu-dashboard/cpu-dashboard.component.html</context>
+          <context context-type="linenumber">56</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit approved="yes" datatype="html" id="7fe0cde0fbcfb34e7fc116339897490407049572">
+        <source>CPU - Frequency</source>
+        <target phase-name="approval-1" state="signed-off">Cpu-frequentie</target>
```

Comment:
CPU is supposed to be written with capital letters: https://nl.wikipedia.org/wiki/Processor_(computer)
```suggestion
        <target phase-name="approval-1" state="signed-off">CPU-frequentie</target>
```